app/views/admin/submissions/index.html.erb

```diff
@@ -44,7 +44,7 @@
             <%= submission.referer %>
           </td>
           <% end %>
-          <td><%= submission.created_at %></td>
+          <td><%= format_time(submission.created_at, submission.form.time_zone.present? ? submission.form.time_zone : current_user.time_zone) %></td>
```

Comment:
viewing submissions by MY timezone vs the Form's timezone is something to consider, UX-wise